src/node-loaders.js

```diff
@@ -31,6 +31,7 @@ var FileSystemLoader = Loader.extend({
                     var watcher = chokidar.watch(p, { ignoreInitial: true });
 
                     watcher.on('all', function(event, fullname) {
+                        fullname = path.resolve(fullname);
```

Comment:
`fullname` is the path that changed according to the path you gave chokidar, `p` in this case. If `p` is absolute, `fullname` will be absolute (I think). If it's relative, it will be relative. If `p` is `views`, `fullname` will be `views/index.html`.

So basically this should work because chokidar's watch patch is relative to `process.cwd()`. If it isn't, you gave it an absolute path.